bin/load-runner-profiles.js

```diff
@@ -56,7 +55,7 @@ var q = async.queue(function profile_task(task, callback) {
       console.log(`stdout: ${stdout}`);
       if (stderr) {
         console.log(`stderr: ${stderr}`);
```

Comment:
I don't know, and I'm not sure we should change it in case something relies on it being output in stdout, even though its an error.
I'll leave this as is